### PR TITLE
build: improve makefile usage + parallelism

### PR DESCRIPTION
**Pull request by dcramer**
Before:

```
time make develop
real	0m23.721s
user	0m16.093s
sys	0m7.922s
```

After:
```
time make develop
real	0m15.069s
user	0m18.554s
sys	0m8.496s
```

### CODE REVIEW

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Changes to build requirements</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: Makefile [hunk 1]
```diff
@@ -1,13 +1,12 @@
+CPUS ?= $(shell sysctl -n hw.ncpu || echo 1)
```
Comment (mattrobenolt) [inline Makefile]: I like how this only works on OSX. I think you'll break it in Travis. Or I guess just fall back to 1.

**Review by mattrobenolt — APPROVED** [review]
ok

File: Makefile (hunk 1 above)
Comment (dcramer) [inline Makefile]: yeah we could probably tweak it, but i was mostly focused on making it faster in dev

**Pull request merged**